Controller/PlaceController.php

```diff
@@ -41,6 +41,26 @@ public function listPlaceListAction( Location $location )
     }
 
     /**
+     * Displays place content.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     * @param string $viewType
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function showPlaceAction( Location $location, $viewType )
+    {
+        $placeHelper = $this->get( 'ezdemo.place_helper' );
+        $contentService = $this->get( 'ezpublish.api.service.content' );
+
+        return $this->get( 'ez_content' )->viewLocation(
+            $location->id,
+            $viewType,
+            true,
+            [ 'relatedImage' => $placeHelper->getImageFromRelatedObjects( $contentService, $location ) ]
```

Comment:
no, i don't see any changes in placeHelper for this pr. just suggesting/asking if it is a good practice mainly.